tests/codegen/riscv-target-abi.rs

```diff
@@ -0,0 +1,20 @@
+//@ revisions:riscv64gc riscv32gc riscv32imac
+
+//@[riscv64gc] compile-flags: --target=riscv64gc-unknown-linux-gnu
+//@[riscv64gc] needs-llvm-components: riscv
+// riscv64gc: !{i32 {{1}}, !"target-abi", !"lp64d"}
```

Comment:
> I am almost certain `// riscv64gc: !{i32 {{1}}, !"target-abi", !"lp64d"}` doesn't check anything.

It works. If I change "lp64d" to "lp64" the test would fail.

Reply:
https://github.com/rust-lang/rust/blob/75fd074338801fba74a8cf7f8c48c5c5be362d08/src/tools/compiletest/src/runtest.rs#L2883-L2892

You're right... that is... quite surprising 😓